packages/caver-transaction/src/transactionHelper/transactionHelper.js

```diff
@@ -0,0 +1,167 @@
+/*
+    Copyright 2020 The caver-js Authors
+    This file is part of the caver-js library.
+
+    The caver-js library is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    The caver-js library is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    along with the caver-js. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+const _ = require('lodash')
+const utils = require('../../../caver-utils/src')
+
+const TX_TYPE_STRING = {
+    TxTypeLegacyTransaction: 'TxTypeLegacyTransaction',
+
+    TxTypeValueTransfer: 'TxTypeValueTransfer',
+    TxTypeFeeDelegatedValueTransfer: 'TxTypeFeeDelegatedValueTransfer',
+    TxTypeFeeDelegatedValueTransferWithRatio: 'TxTypeFeeDelegatedValueTransferWithRatio',
+
+    TxTypeValueTransferMemo: 'TxTypeValueTransferMemo',
+    TxTypeFeeDelegatedValueTransferMemo: 'TxTypeFeeDelegatedValueTransferMemo',
+    TxTypeFeeDelegatedValueTransferMemoWithRatio: 'TxTypeFeeDelegatedValueTransferMemoWithRatio',
+
+    TxTypeAccountUpdate: 'TxTypeAccountUpdate',
+    TxTypeFeeDelegatedAccountUpdate: 'TxTypeFeeDelegatedAccountUpdate',
+    TxTypeFeeDelegatedAccountUpdateWithRatio: 'TxTypeFeeDelegatedAccountUpdateWithRatio',
+
+    TxTypeSmartContractDeploy: 'TxTypeSmartContractDeploy',
+    TxTypeFeeDelegatedSmartContractDeploy: 'TxTypeFeeDelegatedSmartContractDeploy',
+    TxTypeFeeDelegatedSmartContractDeployWithRatio: 'TxTypeFeeDelegatedSmartContractDeployWithRatio',
+
+    TxTypeSmartContractExecution: 'TxTypeSmartContractExecution',
+    TxTypeFeeDelegatedSmartContractExecution: 'TxTypeFeeDelegatedSmartContractExecution',
+    TxTypeFeeDelegatedSmartContractExecutionWithRatio: 'TxTypeFeeDelegatedSmartContractExecutionWithRatio',
+
+    TxTypeCancel: 'TxTypeCancel',
+    TxTypeFeeDelegatedCancel: 'TxTypeFeeDelegatedCancel',
+    TxTypeFeeDelegatedCancelWithRatio: 'TxTypeFeeDelegatedCancelWithRatio',
```

Comment:
Don't we need to have chain data anchoring tx?

Reply:
I added TxTYpeChainDataAnchoring.